source/module_hsolver/hsolver_pw.cpp

```diff
@@ -280,9 +280,18 @@ void HSolverPW<T, Device>::solve(hamilt::Hamilt<T, Device>* pHamilt,
     std::vector<Real> eigenvalues(this->wfc_basis->nks * psi.get_nbands(), 0.0);
     ethr_band.resize(psi.get_nbands(), this->diag_thr);
 
+    // using k-point continuity
+    if (use_k_continuity) {
+        build_k_neighbors();
+    }
+
+    static int count = 0;
```

Comment:
please use a parameter of Psi to control this function, rather than use static variable.